manager_rethink.go

```diff
@@ -62,6 +62,24 @@ func rdbFromPolicy(p Policy) (*rdbSchema, error) {
 	}, err
 }
 
+// NewRethinkManager initializes a new RethinkManager for given session, table name defaults
+// to "policies", policies can default to nil.
+func NewRethinkManager(session *r.Session, table string, policies map[string]Policy) *RethinkManager {
```

Comment:
I just went along how it was dealt in the Tests, but it seems like ColdStart will just overwrite the initial policies, so I am going to fix this up.

While we are at it, panic on empty `table` value? return an error? or is the "ladon" default sensable?